app/main/posts/views/post-toolbar.directive.js

```diff
@@ -23,6 +23,9 @@ function PostToolbarController($scope, $rootScope, Notify, PostLockService, $sta
     $scope.isSaving = LoadingProgress.getSavingState;
     $scope.editMode = editMode;
     $scope.cancel = cancel;
+    $scope.hideOtherActions = hideOtherActions;
+    $scope.showOtherActions = showOtherActions;
```

Comment:
What are they other compared to? Would it be all other actions?

Reply:
Hrm. Maybe its just 'actions' ? Basically we hide the action buttons when the filters are active.